docs/source/app_developers_guide/ubuntu_test_network.rst

```diff
@@ -0,0 +1,441 @@
+.. _proc-multi-ubuntu-label:
+
+Using Ubuntu for a Sawtooth Test Network
+========================================
+
+This procedure describes how to create a network of Sawtooth nodes for an
+application development environment on a Ubuntu platform.
+
+Each host system (physical computer or virtual machine) is a Sawtooth node that
+runs a validator and related Sawtooth components. The first node creates the
+genesis block, which specifies the on-chain network configuration settings.
+Other nodes access those settings when they join the network.
+
+This procedure describes how to configure either PBFT or PoET consensus.
+(Devmode consensus is not recommended for a network.) The initial network must
+include the minimum number of nodes for the chosen consensus:
+
+.. include:: ../_includes/consensus-node-reqs.inc
+
+.. note::
+
+   These instructions have been tested on Ubuntu 16.04 only.
+
+
+.. _about-sawtooth-nw-env-label:
+
+About the Sawtooth Network Environment
+--------------------------------------
+
+This procedure creates a multiple-node Sawtooth network that can be used for
+an application development environment.
+
+.. include:: ../_includes/about-nw-each-node-runs.inc
+
+Like the single-node environment, this environment uses parallel transaction
+processing and static peering. For PBFT consensus, the network must be `fully
+peered` (each node is connected to all other nodes).
+
+The following figure shows an example network that has five nodes:
+
+.. figure:: ../images/appdev-environment-multi-node.*
+   :width: 100%
+   :align: center
+   :alt: Ubuntu: Example Sawtooth network with five nodes
+
+
+.. _prereqs-multi-ubuntu-label:
+
+Prerequisites
+-------------
+
+**Remove data from an existing single node**: To reuse the single test node
+described in :doc:`ubuntu`, stop Sawtooth and delete all blockchain data
+and logs from that node.
+
+1. If the first validator node is running, stop the Sawtooth components
+   (validator, REST API, and transaction processors), as described
```

Comment:
I think this is missing the consensus engine. It should be stopped as well.

Reply:
Fixed.